README.md

```diff
@@ -1,31 +1,35 @@
 # CompLOINC LinkML
 
-Setup:
-python 3.9.7
-pip install -r requirements.txt
+## Prerequisities
+1. Python 3.9.7+
+2. Required LOINC Files
```

Comment:
@putmantime I know we should probably use Git-LFS, but I created a GoogleDrive folder to hold these files for now: https://drive.google.com/drive/folders/1xLHmUJsF4yigH0vj-xwc3OfLXTbSzoWx?usp=sharing